src/kubernetes/skipper/skipper-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: skipper
-        image: springcloud/spring-cloud-skipper-server:2.2.1.BUILD-SNAPSHOT
+        image: springcloud/spring-cloud-skipper-server:2.3.0.BUILD-SNAPSHOT
```

Comment:
Is it possible to inject the ${skipper.version} defined from the POM file into the template files?

Reply:
That's exactly what happens when this file is generated from a template. This change happened as pom is using skipper 2.3.0.BUILD-SNAPSHOT

Reply:
oh, I see. I get it now.